docs/application/native/guides/TOC.md

```diff
@@ -145,6 +145,7 @@
 ## [Overview](security/overview.md)
 ## [Secure Key Management](security/secure-key.md)
 ## [Privilege Information](security/privilege.md)
+## [Privacy-related Permissions](security/privacy-related-permissions.md)
```

Comment:
It looks like we don't have privacy-related-permissions.md file. This link is broken.

Reply:
Done